pallets/external-validators-rewards/src/lib.rs

```diff
@@ -271,70 +271,82 @@ pub mod pallet {
-                        rewards_merkle_root: utils.rewards_merkle_root,
-                        token_id,
-                    };
-
-                    let channel_id: ChannelId = snowbridge_core::PRIMARY_GOVERNANCE_CHANNEL;
-
-                    let outbound_message = Message {
-                        id: None,
-                        channel_id,
-                        command: command.clone(),
-                    };
-
-                    // Validate and deliver the message
-                    match T::ValidateMessage::validate(&outbound_message) {
-                        Ok((ticket, _fee)) => {
-                            let message_id = ticket.message_id();
-                            if let Err(err) = T::OutboundQueue::deliver(ticket) {
-                                log::error!(target: "ext_validators_rewards", "OutboundQueue delivery of message failed. {err:?}");
-                            } else {
-                                Self::deposit_event(Event::RewardsMessageSent {
-                                    message_id,
-                                    rewards_command: command,
-                                });
-                            }
-                        }
-                        Err(err) => {
-                            log::error!(target: "ext_validators_rewards", "OutboundQueue validation of message failed. {err:?}");
-                        }
-                    }
+            let Some(utils) = Self::generate_era_rewards_utils(era_index, None) else {
+                // Unreachable, this should never happen as we are sending
+                // None as the second param in Self::generate_era_rewards_utils.
+                log::error!(
+                    target: "ext_validators_rewards",
+                    "Outbound message not sent for era {:?}!",
+                    era_index
+                );
+                return;
+            };
 
-                    frame_system::Pallet::<T>::register_extra_weight_unchecked(
-                        T::WeightInfo::on_era_end(),
-                        DispatchClass::Mandatory,
-                    );
-                } else {
-                    // Unreachable, this should never happen as we are sending
-                    // None as the second param in Self::generate_era_rewards_utils.
-                    log::error!(
-                        target: "ext_validators_rewards",
-                        "Outbound message not sent for era {:?}!",
-                        era_index
-                    );
+            let tokens_inflated = T::EraInflationProvider::get();
+
+            if tokens_inflated.is_zero() {
+                log::error!(target: "ext_validators_rewards", "Not sending message because tokens_inflated is 0");
+                return;
+            }
+
+            if utils.total_points.is_zero() {
+                log::error!(target: "ext_validators_rewards", "Not sending message because total_points is 0");
```

Comment:
Can we have this documented somewhere? Not sure if at the top of this function or where, but basically "Sends a message to ethereum, but only if there are rewards to distribute".

Or maybe in the Command::ReportRewards fields? Say that the rewards will never be 0?